docs/user-guide/adding-widgets.md

```diff
@@ -33,36 +33,41 @@ In dashboards, selecting the Map type widget, the following panel appears:
 
 Here the user can:
 
-* Go back <img src="../img/button/back.jpg" class="ms-docbutton"/> to widget type selection
+* Go back to widget type selection through the <img src="../img/button/back.jpg" class="ms-docbutton"/> button 
 
 * Search for a map by writing its title
 
-* Select a map (mandatory in order to move forward)
+* Select a map from the list of maps (mandatory in order to move forward)
 
-* Move forward to the next step <img src="../img/button/next.jpg" class="ms-docbutton"/>
+* Move forward to the next step through the <img src="../img/button/next.jpg" class="ms-docbutton"/> button
 
-The next panel is similar to the following (in this case an empty map was selected):
+Once a map has been selected, a panel opens and the layers present in the map are displayed in the preview and listed in the layer list:
 
-<img src="../img/adding-widgets/wid-map-options.jpg" class="ms-docimage"  style="max-width:400px;"/>
+<img src="../img/adding-widgets/wid-layers-list.jpg" class="ms-docimage"  style="max-width:400px;"/>
 
-Now the user is allowed to add layers to the map through the <img src="../img/button/+++.jpg" class="ms-docbutton"/> button:
+!!!note
+    If the **Empty Map** is selected the user can:
 
-<img src="../img/adding-widgets/wid-add-layer.gif" class="ms-docimage"  style="max-width:400px;"/>
+    * Create a widget map using an empty map
 
-Once a layer is added to the map widget, it will be displayed in preview and in layers list:
+    * Add layers to the map through the <img src="../img/button/+++.jpg" class="ms-docbutton"/> button, as follows: <img src="../img/adding-widgets/wid-add-layer.gif" class="ms-docimage"  style="max-width:400px;"/>
 
-<img src="../img/adding-widgets/wid-layers-list.jpg" class="ms-docimage"  style="max-width:400px;"/>
+On the **Configure map options** panel the user can toggle the layer visibility and set layers transparency, as explained in the [Display options](toc.md#display-options-in-panel) section. Furthermore the user can manage the layer with the new buttons present on the layer toolbar by selecting the layer on the layers list.
```

Comment:
On the **Configure map options** panel the user can toggle the layer visibility and set layers transparency, as explained in the [Display options](toc.md#display-options-in-panel) section. Furthermore, the user can manage the layer with the new buttons present on the layer toolbar by selecting the layer on the layers list.